File: oldandroidsupport/src/main/java/com/tq/app/libs/callback/OnAdapterCallback.java
/*
 * ******************************************************************************
 *  Copyright Ⓒ 2016. TrinhQuan. All right reserved
 *  Author: TrinhQuan. Created on 2016/12/20
 *  Contact: trinhquan.171093@gmail.com
 * ******************************************************************************
 */

package com.tq.app.libs.callback;

import com.tq.app.libs.data.IDataWrapper;

public interface OnAdapterCallback {
    void onAdapterEventPerformed(String adapterID, int eventID, IDataWrapper data);
}

File: kotlinandroidsupport/src/main/java/com/tq/klibs/callback/OnAdapterCallback.kt
/*
 * ******************************************************************************
 *  Copyright Ⓒ 2016. TrinhQuan. All right reserved
 *  Author: TrinhQuan. Created on 2016/12/20
 *  Contact: trinhquan.171093@gmail.com
 * ******************************************************************************
 */

package com.tq.klibs.callback

import android.os.Bundle

interface OnAdapterCallback {

    fun onAdapterEvent(eventId: Int, data: Bundle?)
}
